src/vcflite/README.md

```diff
@@ -0,0 +1,90 @@
+# vcflite
+
+A very early stab at implementing VCF querying and filtering using [noodles](https://crates.io/crates/noodles) and [SQLite](https://www.sqlite.org).
+
+The goal is to be simple and performant: After all SQL is highly specialized in set operations, and Rust with `noodles` provides a performant environment for handling VCF files.
+
+
+## Installation
+
+The easiest way to install `vcflite` is to use the Docker image:
```

Comment:
the gavrie/vcflite:0.2.1 is actually the docker that built from the src/vcflite/Dockerfile?
if yes, maybe we don't want to stick an hard coded image here (I don't think we remember to update this readme for every change in the Docker) but to build it from the Dockerfile (as we do in the action)